org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/corrections/proposals/AssignToVariableAssistProposal.java

```diff
@@ -89,7 +90,7 @@
 	private VariableDeclarationFragment fExistingFragment;
 
 	public AssignToVariableAssistProposal(ICompilationUnit cu, int variableKind, ExpressionStatement node, ITypeBinding typeBinding, int relevance) {
```

Comment:
This constructor is never been called. Shall I remove it or keep it? @testforstephen as the original author, what's your suggestion?